### DISCUSSION TITLE
Go to specific Workspace by Config

### DISCUSSION THREAD

**Discussion by dorsiflexion** [i3/i3#5453] [Q&A] [answered]
Hey,
What determines which workspace is focused after starting?
Can I configure i3 to automatically go to/focus a specific empty workspace when starting? How?

Thanks in advance.

**Comment by orestisfl** [comment]
It's not very intuitive but the very first workspace `bindsym` determines the starting workspace IIRC.

So 
```
bindsym x workspace some-name
bindsym $mod+1 workspace number 1
...
```
should make `some-name` the initial workspace

**Comment by dorsiflexion** [comment, marked as answer]
Thanks!
In addition, I just found the answer how to jump to a workspace by config:
`exec --no-startup-id i3 workspace number 'workspace name'`

**Comment by DeedeeOlesonFnt** [comment]
By default, i3wm focuses on the workspace that was active before you last shut down your i3 session. However, you can configure i3 to start on a specific workspace when you launch it.

**Reply by orestisfl** [reply]
That's incorrect, i3 has no memory of which workspace was focused in previous sessions
